src/app/zap-templates/templates/chip/CHIPClusters-src.zapt

```diff
@@ -15,7 +15,7 @@ namespace Controller {
 
 // {{asCamelCased name false}} Cluster Commands
 {{#chip_server_cluster_commands}}
-CHIP_ERROR {{asCamelCased clusterName false}}Cluster::{{asCamelCased name false}}(Callback::Callback<> * onCompletion{{#chip_server_cluster_command_arguments}}, {{chipType}} {{asCamelCased label}}{{/chip_server_cluster_command_arguments}})
+CHIP_ERROR {{asCamelCased clusterName false}}Cluster::{{asCamelCased name false}}(Callback::Callback<> * onCompletion{{#chip_server_cluster_command_arguments}}, {{chipType}} {{asCamelCased label}}{{#if (isByteString type)}}, uint32_t {{asCamelCased label}}Len{{/if}}{{/chip_server_cluster_command_arguments}})
```

Comment:
We can do better here. The whole point to `chip_server_cluster_command_arguments` was to try to minimise the amount of additional code in the template.

It is OK to do  {{#if ...}}...{{/if}}` blocks inside for some exceptional cases that should not be replicated all over the uses of `chip_server_cluster_command_arguments` but for this case what we should do instead if to fix the `chip_server_cluster_command_arguments` function code in `helper.js` to generate 2 arguments for `BYTE_STRING` instead of 1. Then it will be transparent for all consumers of the API.

Reply:
Let's trace this in #4503